File: courses/agentscope-agent-engineering/demos/basics-03-tools-mcp-lab/mock_mcp.py
"""本地 Mock MCP 工具，用于无 MCP Server 时验收 Toolkit 注册。"""


def search_kb(query: str) -> str:
    """检索 CorpAssist 知识库（Mock）。"""
    return f"[mock] 关于「{query}」：请参考《员工手册》第 3 章。"

File: courses/agentscope-agent-engineering-java/demos/basics-03-tools-mcp-lab/MockMcp.java
/** 模拟 MCP Server list_tools 返回。 */
public class MockMcp {
    public static String searchKb(String query) {
        return "KB 命中: 「" + query + "」年假按工龄累计，详见 HR-2026-01";
    }
}
